java/src/main/java/com/kevinthorley/csi280/project01/ClassRepo.java

```diff
@@ -15,11 +18,17 @@ public ClassRepo() {
   }
 
   public Section findById(String id) {
-    // TODO: find a section by its id
+    for (int i = 0; i < sections.size(); i++)
+    {
+    	if (sections[i].getId() == id)
```

Comment:
Correct syntax for accessing an element of a list is sections.get(i). I modified this statement and the following statement and was able to get the tests to pass.